src/docs/user/references/Notification-service.adoc

```diff
@@ -353,22 +353,16 @@ INFO, WARNING, ERROR, CRITICAL. The default severity is INFO.
 [cols="2,2,2,5,5"]
 |===
 | *Type* | *Readable name* | *Severity* | *Message* | *Description*
-| `ASSOCIATION_FAILED`
-| Association failed
+.3+| `ASSOCIATION_FAILED`
+.3+| Association failed
 | CRITICAL
 | The server was unable to retrieve the resource Native_Task_Windows in the bucket basic-examples
 located \https://{HOST}/catalog/buckets/basic-examples/resources/Native_Task_Windows/raw.
 Changing the status of the association 152 of the calendar every_minute to FAILED.
 | A <<_glossary_catalog, *Catalog*>> resource is not found, the association changes to FAILED status
-| `ASSOCIATION_FAILED`
-| Association failed
-| ERROR
-| The association 52 of the workflow Variables_Propagation to the calendar every_monday had a failed execution
-, as it is configured, the association status is set to FAILED
+.2+| ERROR
+| The association 52 of the workflow Variables_Propagation to the calendar every_monday had a failed execution, as it is configured, the association status is set to FAILED
```

Comment:
, as it is configured => As it is configured,
even though we could not probably edit this probably

Reply:
Absolutely This needs to be changed in job planner service